### DISCUSSION TITLE
Is you were last "verified as null" the same as "you were last verified on blank"

### DISCUSSION THREAD

**Discussion by GCustom** [community/community#113340] [GitHub Education] [answered]
### Hi everyone,

In the pinned FAQ page, it says to raise a thread if you get the blank message, and I'm not sure if the null message is the same case. Here's my screenshot, just in case I should still be making a thread. I was initially rejected due to my billing profile not having any name, thus failing to match the name on my documents.

![Screenshot 2024-03-19 at 16 38 43](https://github.com/community/community/assets/12734177/3cbd11d7-953b-42eb-b315-d23b4f781e5d)

**Comment by SebastienXia** [comment]
Hello, @GCustom 

> I invite you to read this [here](https://github.com/orgs/community/discussions/111352#discussion-6342154)

You should have all informations about your issue. :) enjoy

**Reply by GCustom** [reply]
Yes, that was the post that prompted me to create this post; when I read it, it didn't specify if null was different from blank.

**Comment by samus-aran** [comment, marked as answer]
Hi @GCustom 👋 

If you are receiving that message and your top most recent application status is "Approved" in green then you don't need to do anything else except wait for your academic benefits to be granted. This usually takes 72 hours since when you were Approved.

The "null" is a misnomer as it would usually state you as a student or teacher once you receive your academic benefits - but since you are still in the activation period (72 hours) it doesn't display anything 😅 

Hope this helps! 👍

**Reply by samus-aran** [reply]
P.S @GCustom 

I've updated the FAQ to include this [You were last verified as a "null" on...](https://github.com/orgs/community/discussions/111352#null-last-verified-on-date) section to help others 👍

**Reply by GCustom** [reply]
Thanks. I wasn't trying to be a pest. The update will remove any uncertainty for anyone else with that thought pattern, as both conditions are now addressed.

**Reply by samus-aran** [reply]
>  I wasn't trying to be a pest. 

You were certainly not a pest! An absolute star ⭐ 🙇 Thank you for raising it 👍
